trino-aws-proxy/src/main/java/io/trino/aws/proxy/server/TrinoAwsProxyServerModule.java

```diff
@@ -147,8 +151,13 @@ protected void setup(Binder binder)
             return AssumedRoleProvider.NOOP;
         });
 
+        // RemoteS3 binder
+        configBinder(binder).bindConfig(RemoteS3Config.class);
```

Comment:
This shouldn't be necessary. `TrinoAwsProxyServerBinding` should bind it already.

Reply:
I think this is right, this is the config that decides the plugin type

Reply:
I believe `TrinoAwsProxyServerBinding` already binds the config

Reply:
It's true that explicit config binding is not needed when using `conditionalModule`, I wanted to be consistent with the rest of this class.